draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1992,7 +1992,12 @@ uCDN-A                         dCDN-B                         dCDN-C
         "type": "hls"
       }
     }
-  ],
+   ],
+
```

Comment:
Looks like a superfluous new line that could be removed.

Reply:
Fixed